packages/docs/src/page-configs/services/breakpoints/code-examples/helpers.ts

```diff
@@ -0,0 +1,12 @@
+export const helpersCodeTemplate = `
+<template>
+  <h1 v-if="$vaBreakpoints.mdUp">Title</h1>
+</template>
+`
+
+export const helpersCodeSetup = `
+setup: () => {
+  const breakpoints = inject('$vaBreakpoints')
```

Comment:
I think the hook will be more convenient here
```suggestion
  const breakpoints = useBreakpoints()
```